site/src/App.style.js

```diff
@@ -85,6 +86,10 @@ export const Main = styled.main`
 
   .form {
     margin: 4rem auto;
+    h2{
+      font-size: 2.7rem;
+      margin-left: 2rem;
```

Comment:
Instead of adding this margin. Reduce the `width` of `main` to `75%` and reduce the margin of `.Form-page` to `0 1rem`.